.github/workflows/functional_test.yml

```diff
@@ -64,42 +64,55 @@ jobs:
             driver: docker
             cruntime: docker
             os: ubuntu-22.04
+            runner: ubuntu-22.04
             test-timeout: 15m 
           - name: docker-containerd-ubuntu-22.04-x86_64
             driver: docker
             cruntime: containerd
             extra-start-args: --container-runtime=containerd
             os: ubuntu-22.04
+            runner: ubuntu-22.04
             test-timeout: 15m 
           - name: docker-containerd-rootless-ubuntu-22.04-x86_64
             driver: docker
             cruntime: containerd
             os: ubuntu-22.04            
+            runner: ubuntu-22.04
             extra-start-args: --container-runtime=containerd --rootless
             rootless: true
             test-timeout: 15m 
           - name: podman-docker-ubuntu-24.04-x86_64
             driver: podman
             cruntime: docker
             os: ubuntu-24.04
+            runner: ubuntu-24.04
             test-timeout: 15m
           - name: baremetal-docker-ubuntu-22.04-x86_64
             driver: none
             cruntime: docker
             os: ubuntu-22.04
+            runner: ubuntu-22.04
             test-timeout: 7m
           - name: qemu-docker-macos-13-x86_64
             driver: qemu
             cruntime: docker
             os: macos-13
+            runner: macos-13
             extra-start-args: --network socket_vmnet
             test-timeout: 50m
           - name: vfkit-docker-macos-13-x86_64
             driver: vfkit
             cruntime: docker
             os: macos-13
+            runner: macos-13
             extra-start-args: --network vmnet-shared
             test-timeout: 50m            
+          - name: docker-docker-ubuntu-arm64
+            driver: docker
+            cruntime: docker
+            os: ubuntu-22.04
+            runner: [self-hosted, arm64]
```

Comment:
we would need to see if we can make that Only one of the Matrix items be required to have ok-to-test label and the test be  as it is, can you see if thats possible?


feel free to make a Dummy Repo just to check that in a Dummy repo with a dummy workflow on your own repo